src/components/fold-media.css

```diff
@@ -0,0 +1,7 @@
+.FoldMedia .AttributionEditor-metadata {
+  top: -75px;
+}
```

Comment:
If the app is animating the cover image, we'll need an option when mounting to hide the fold cover, and let the app take care of the overlap. That will complicate the new post flow though.